lib/hexpm/repository/package_reports.ex

```diff
@@ -32,25 +40,110 @@ defmodule Hexpm.Repository.PackageReports do
     |> Repo.one()
     |> PackageReport.change_state(%{"state" => "accepted"})
     |> Repo.update()
+
+    report = PackageReport.get(report_id)
+
+    report =
+      PackageReport.get(report_id)
+      |> Repo.one()
+
+    email_user_about_state_change(report, report.author)
+
+    Enum.each(
+      Owners.all(report.package),
+      fn owner ->
+        owner = Hexpm.Repo.preload(owner, user: [])
+        email_user_about_state_change(report, owner.user)
+      end
+    )
+
+    Enum.each(
+      Users.get_by_role("moderator"),
+      fn user ->
+        email_user_about_state_change(report, user)
+      end
+    )
   end
 
   def reject(report_id, comment) do
     PackageReport.get(report_id)
     |> Repo.one()
     |> PackageReport.change_state(%{"state" => "rejected"})
     |> Repo.update()
+
+    report =
+      PackageReport.get(report_id)
+      |> Repo.one()
+
+    email_user_about_state_change(report, report.author)
+
+    Enum.each(
+      Owners.all(report.package),
+      fn owner ->
+        owner = Hexpm.Repo.preload(owner, user: [])
+        email_user_about_state_change(report, owner.user)
+      end
+    )
```

Comment:
Package owners do not know about reports until the reported has been accepted so no need to email them if it has been rejected.